### DISCUSSION TITLE
I can't upgrade rappasoft/laravel-livewire-tables from 0.3.3  to a higher version.

### DISCUSSION THREAD

**Discussion by fikoulli** [rappasoft/laravel-livewire-tables#262] [Q&A] [answered]
Dear Rappasoft,

First, thx for your efforts and this great component.
One question plz :

I'm using :
- laravel-boilerplate-7.2.5 
- PHP 7.4.13 (cli)
- composer 2
- I have this in composer.json : 
        "rappasoft/laravel-livewire-tables": "^0.4.0",
        
As u can see, I'm in version 0.4.0

- If i run this command : 
    composer update  -W --dry-run
  laravel-livewire-tables in not on the list of the upgrades :(
  
  To be sure, the command : composer update rappasoft/laravel-livewire-tables -W
  gives : Nothing to install, update or remove

Q : Plz, how to upgrade this component ?

**Comment by rappasoft** [comment, marked as answer]
Thats because ^0.4.0 is the last version < 1.0 which is what you're telling composer to look for. 1.0 is completely different and will break your project unless you build your components with the new 1.0 methods. If you want to get composer to update to 1.0 you should use `"rappasoft/laravel-livewire-tables": "^1.0"`

**Reply by fikoulli** [reply]
Thx & good luck
